code/src/Components/GetLast-Post.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react'
+
+const showTime= (createdAt)=>{
+  const today = new Date()
+  const messageDate = new Date (createdAt)
+  
+  let convertDate= Math.abs(today- messageDate) /1000
+  
+  const hours= Math.floor(convertDate/ 3600) %24;
+  convertDate -= hours * 3600
+  
+  const minutes= Math.floor(convertDate/60)% 60;
+  convertDate -= minutes * 60;
+  
+  if(hours !== 0){
+    return(`${hours} hours ago`)
+  }  else{
+    return (`${minutes} minutes ago`)
+  }
+}
```

Comment:
Remember that NPM packages like moment.js takes good care of this stuff for you. Nevertheless, this code seems to work fine :D